src/style.css

```diff
@@ -1 +1,63 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+}
+
+.search-bars {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  margin-top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+}
```

Comment:
You are styling the `.search-bars` using tag names which is against the checklist. Style all elements using `.classes` and if needed with `:pseudo-classes`, `pseudo-elements` and `[attributes]`.